distribution/debian/install.sh

```diff
@@ -159,23 +260,48 @@ Restart=on-failure
 [Install]
 WantedBy=multi-user.target
 EOF
+sleep 2
+echo ""
+echo -e "New service file created!"
 
 # Start the App
-echo "Service file created. Attempting to start the app"
+echo ""
+echo -e "${brown}[${app^}]${reset} is attempting to start, this may take a few seconds..."
 systemctl -q daemon-reload
 systemctl enable --now -q "$app"
+sleep 3
+
+# Check if the service is up and running
+echo ""
+echo "Checking if the service is up and running..."
+# Loop to wait until the service is active
+while ! systemctl is-active --quiet "$app"; do
+    sleep 1
+done
+echo ""
+echo -e "${brown}[${app^}]${reset} installation and service start up is complete!"
 
-# Finish Update/Installation
+# Finish Installation
 host=$(hostname -I)
 ip_local=$(grep -oP '^\S*' <<<"$host")
 echo ""
-echo "Install complete"
-sleep 10
+echo -e "Attempting to check for a connection at http://$ip_local:$app_port..."
+sleep 3
 STATUS="$(systemctl is-active "$app")"
 if [ "${STATUS}" = "active" ]; then
-    echo "Browse to http://$ip_local:$app_port for the ${app^} GUI"
+    echo ""
+    echo "Successful connection!"
+    echo ""
+    echo -e "Browse to ${green}http://$ip_local:$app_port${reset} for the GUI."
+    echo ""
+    echo "Script complete! Exiting now!"
+    echo ""
 else
-    echo "${app^} failed to start"
+    echo ""
+    echo -e ${red}"${app^} failed to start."${reset}
+    echo ""
+    echo "Please try again. Exiting script."
+    echo
```

Comment:
using the check against `systemctl is-active` isn't  necessarily checking for a successful connection, only an active service.
This part may be misleading.
Also not sure what the difference between this check and the previous is-active check is.